src/stories/components/cu-activity-table/cu-activity-table.placeholder.tsx

```diff
@@ -37,9 +39,12 @@ export const ActivityPlaceholder = (props: { hasFilter: boolean }) => {
             buttonType={ButtonType.Secondary}
           />
         )}
-        <a href={SUBMIT_EXPENSES_URL} target="_blank">
-          <CustomButton style={{ minWidth: '250px' }} label="Submit Expenses Now" buttonType={ButtonType.Primary} />
-        </a>
+        <CustomButton
```

Comment:
can this be a `Link` (from nextjs) to navigate instead a button with a `onClick` event?

Reply:
You are right, in next refactoring, we will extend the component to have both functions